lib/src/op_attribute_sanitizer.dart

```diff
@@ -199,9 +199,10 @@ class OpAttributes {
 typedef UrlSanitizerFn = String? Function(String url);
 
 class OpAttributeSanitizerOptions {
-  OpAttributeSanitizerOptions({this.urlSanitizer});
+  OpAttributeSanitizerOptions({this.urlSanitizer, this.argbHexColors = true});
 
   UrlSanitizerFn? urlSanitizer;
+  bool argbHexColors;
```

Comment:
My intention was to have a switch here whether 8 digit hex colors were allowed or not. 

Secondly, since CSS hex colors are only RGBA (https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color), saying "argb..." here is confusing. I now understand that it is ARGB _input_ color vs. RGBA. But I don't think this is a needed case. CSS is always RGBA.

I'd rather have this be an `allow8DigitHexColors` switch. It should be false by default for backwards compatibility. If it is false, the sanitizer should reject 8 digit hex colors.

Please also add a comment to the field and update README.md.